app/controllers/items_controller.rb

```diff
@@ -141,8 +141,14 @@ def source_id
   end
 
   def catkey
-    @object.catkey = params[:new_catkey].strip
-    save_and_reindex
+    object_client = Dor::Services::Client.object(@object.pid)
+    dro = object_client.find
+    updated_identification = dro.identification.new(
+      catalogLinks: [{ catalog: 'symphony', catalogRecordId: params[:new_catkey].strip }]
```

Comment:
Wouldn't this step on all existing catalogLinks? Existing source id?

Reply:
Yes, that's the same behavior as the existing code: https://github.com/sul-dlss/argo/pull/2236/files#diff-2eb2bf110a48bee6c79d3bbb377a549eL144

Reply:
Where is `@object.catkey` implemented?

Reply:
it's part of the dor-services gem.
https://github.com/sul-dlss/dor-services/blob/4cf2fc6503060656a3485856109501d7b3a79011/lib/dor/models/abstract.rb#L79

Reply:
It looks like we will have to store the previous catkeys though so that we can create blank catalog records for old catkeys in the release workflow. https://github.com/sul-dlss/dor-services/blob/master/lib/dor/datastreams/identity_metadata_ds.rb#L128

Reply:
Turns out we don't because dor-services-app is calling the `catkey=` method for us. https://github.com/sul-dlss/dor-services-app/blob/master/app/services/cocina/object_updater.rb#L75